cvat/apps/engine/task.py

```diff
@@ -257,7 +257,7 @@ def _download_data(urls, upload_dir):
         job.meta['status'] = '{} is being downloaded..'.format(url)
         job.save_meta()
 
-        response = requests.get(url, stream=True)
+        response = requests.get(url, stream=True, allow_redirects=True)
```

Comment:
@azhavoro , do you think it introduces any security risks? Will the code above validate redirected URL?